build.sh

```diff
@@ -16,12 +37,30 @@ echo ""
 echo "Which signing identity do you want to use?"
 echo "1. SURFnet B.V. (ZYJ4TZX4UU)"
 echo "2. Egeniq (E85CT7ZDJC)"
-echo "3. Other"
-read -p "0-9?" choice
+echo "3. Enter own Team(This isn't just a random name, but it must be exact as in their signing identity): "
+read -p "1-3?" choice
+
+
+# Enter custom Team ID and Team Name.
+if [ "$choice" == 3  ]
+then
```

Comment:
Add line: `echo "Your Team ID and Team Name must match exactly with the signing identity in your keychain."`

Reply:
Done